osc-server/src/main/java/org/osc/core/broker/service/validator/DeleteDistributedApplianceRequestValidator.java

```diff
@@ -50,6 +57,18 @@ public DistributedAppliance validateAndLoad(BaseDeleteRequest request) throws Ex
                             + request.getId()
                             + " is not marked for deletion and force delete operation is applicable only for entries marked for deletion.");
         }
+        
+        for (VirtualSystem vs : da.getVirtualSystems()) {
+            if (!this.apiFactoryService.supportsNeutronSFC(vs)) {
+             // if first virtual system is not neutronSFC break...
+                break; 
+            }
+            if (vs.getServiceFunctionChains().size() > 0) {
+                throw new VmidcBrokerValidationException("Distributed Appilance with ID " + request.getId()
```

Comment:
can we make the exception message more explicit? specify the sfc name/id which they need to unbind and from which virtual system etc

Something along the lines of(without the abbreviations)
Cannot delete DA as one of the VS'es VSA is being referenced by SFCA